tests/unittests/test_datasource/test_ec2.py

```diff
@@ -616,19 +746,19 @@ class TestConvertEc2MetadataNetworkConfig(test_helpers.CiTestCase):
 
     def setUp(self):
         super(TestConvertEc2MetadataNetworkConfig, self).setUp()
-        self.mac1 = '06:17:04:d7:26:09'
+        self.mac1 = '06:17:04:D7:26:09'
```

Comment:
what caused this change?

Reply:
accidental and manual, should be left lowercase as we are validating the upper to lower in other tests below.